doc/api-extensions.md

```diff
@@ -1257,3 +1257,8 @@ with `ovn.ingress_mode=routed`.
 ## projects\_limits\_instances
 Adds `limits.instances` to the available project configuration keys. If set, it
 limits the total number of instances (VMs and containers) that can be used in the project.
+
+## instances
```

Comment:
We were unsure whether or not we should include an entry here, as well as what the header for it should be, should we also include a new entry in shared/version/api.go?

Reply:
Yes please do include the entry in shared/version/api (as part of the same commit that adds the modification to this file).

Also the API extension name `instances` is too vague as it does not explain what is being added, suggest something like `instance_bulk_state_change`.